From 53b4963b62fa0ad1056ffd982b6e997f466f4df9 Mon Sep 17 00:00:00 2001
From: Anas Nashif <anas.nashif@intel.com>
Date: Sun, 9 Dec 2012 09:51:23 -0800
Subject: [PATCH 1/9] tizen-rpm: 2 useful macro for RPM

- Add %install_service macro
- Define %_unitdir_user macro for user session units

Change-Id: Idc7f5c392c96981d95420b0d747eaf28964b2d02
---
 src/core/macros.systemd.in | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/src/core/macros.systemd.in b/src/core/macros.systemd.in
index bea6ef1..0be9bd0 100644
--- a/src/core/macros.systemd.in
+++ b/src/core/macros.systemd.in
@@ -21,6 +21,7 @@
 
 %_unitdir @systemunitdir@
 %_userunitdir @userunitdir@
+%_unitdir_user @userunitdir@
 %_presetdir @systempresetdir@
 %_udevhwdbdir @udevhwdbdir@
 %_udevrulesdir @udevrulesdir@
@@ -107,3 +108,9 @@ echo %{?*} | systemd-sysusers - >/dev/null 2>&1 || : \
 %binfmt_apply() \
 @rootlibexecdir@/systemd-binfmt %{?*} >/dev/null 2>&1 || : \
 %{nil}
+
+
+%install_service() \
+mkdir -p %{buildroot}/%{_unitdir}/%{1} \
+ln -s ../%{2}  %{buildroot}/%{_unitdir}/%{1}/%{2}  \
+%{nil}
-- 
1.8.4.5

